COMMANDS.md

```diff
@@ -81,7 +81,7 @@ Command Syntax | Plugin class | Description | Working?
 `!turl <url>` | TinyURL | Create a short link to the given url with [TinyURL](http://tinyurl.com/). | Yes
 `!ud <query>` | UrbanDictionary | Look up the given query on [UrbanDictionary](http://www.urbandictionary.com/). | Yes
 `!ver <nick>` | CTCPVersion | Send a CTCP VERSION request to the given nick. | Yes
-`!w <location>` | Weather | Get the weather for the given location from [APIXU](https://www.apixu.com/). | Yes
+`!w <location>` | Weather | Get the weather for the given location from [WEATHERSTACK](https://weatherstack.com). | Yes
```

Comment:
Nit: `WEATHERSTACK` -> `WeatherStack`?

Reply:
ah I guess it was only caps because APIXU was caps on the site too. and I just lazily replaced it without thinking about it